rir/src/interpreter/interp.cpp

```diff
@@ -2190,7 +2190,7 @@ SEXP evalRirCode(Code* c, InterpreterInstance* ctx, SEXP env,
             SEXP val = ostack_top(ctx);
             assert(TYPEOF(val) == INTSXP);
             int i = INTEGER(val)[0];
-            if (MAYBE_SHARED(val)) {
+            if (!NO_REFERENCES(val)) {
```

Comment:
Can you flip the condition? Double negatives confuse me. :)

Reply:
who wouldn't not hate them too?